content/azure-pipeline-blueprints-explore-qa-scans.md

```diff
@@ -0,0 +1,217 @@
+Title: Azure DevOps Pipeline Blueprints - Exploring the QA Scan template9s)
+Date: 2025-01-XX
+Category: Posts
+Tags: azure-devops, pipelines, engineering
+Slug: azure-pipeline-blueprints-explore-qa-scans
+Author: Willy-Peter Schaub
+Summary: Drilling Deeper: A Closer Look at the Blueprint Quality Assurance (QA) Scan Templates.
+
+We continue our deep dive into the details of our [AzureDevOps.Automation.Pipeline.Templates.v2](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2) open source blueprint repo. Today we are going to explore the [templates/qa/azure-pipeline-qa-scans.yml](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2/blob/master/templates/qa/azure-pipeline-qa-scans.yml) and [templates/qa/qa-scans-cd.yml](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2/blob/master/templates/qa/qa-scans-cd.yml) templates, designed to run the quality assurance validations.
+
+> ![drilling](../images/azure-pipeline-blueprints-explore-qa-scans-1.png)
+
+Also refer to:
+
+- [Azure DevOps Pipeline Blueprints - Exploring the start template](/azure-pipeline-blueprints-explore-start.html).
+- [Azure DevOps Pipeline Blueprints - Exploring the version template](/azure-pipeline-blueprints-explore-version.html).
+- [Azure DevOps Pipeline Blueprints - Exploring the info template](/azure-pipeline-blueprints-explore-info.html).
+- [Pipelines - Why bother and what are our nightmares and options?](/why-pipelines-part1.html) blog series.
+
+---
+
+# Today's topics - azure-pipeline-qa-scans.yml and qa-scans-cd.yml
+
+Today we peek into the [templates/qa/azure-pipeline-qa-scans.yml](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2/blob/master/templates/qa/azure-pipeline-qa-scans.yml) template which is triggered be the continuous delivery (cd) template and has one purpose, to trigger the [templates/qa/qa-scans-cd.yml](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2/blob/master/templates/qa/qa-scans-cd.yml) and in fiture, other QA templates.
```

Comment:
"triggered be the" should be "triggered by the"

Reply:
Will fix in publication PR. Thanks!